lib/vmdb/plugins.rb

```diff
@@ -8,6 +8,11 @@ class Plugins
     def initialize
       @registered_automate_domains = []
       @vmdb_plugins = []
+
+      Rails.application.railties.each do |railtie|
+        next unless railtie.class.name.start_with?("ManageIQ::Providers::") || railtie.try(:vmdb_plugin?)
+        register_vmdb_plugin(railtie)
+      end
```

Comment:
If `reload!` is reinstantiating this class, then won't it reregister things like settings paths, automate domains and such?  This seems dangerous.